cmd/crossplane/main.go

```diff
@@ -59,7 +65,11 @@ func main() {
 		extCmd = app.Command("stack", "Perform operations on stacks")
 
 		// stack manage - adds the stack manager controllers and starts their reconcile loops
-		extManageCmd = extCmd.Command("manage", "Manage stacks (run stack manager controllers)")
+		extManageCmd                     = extCmd.Command("manage", "Manage stacks (run stack manager controllers)")
+		extManageTenantKubeconfig        = extManageCmd.Flag("tenant-kubeconfig", "The absolute path of the kubeconfig file to Tenant Kubernetes instance. Setting this will activate host aware mode of Stack Manager").String()
+		extManageHostControllerNamespace = extManageCmd.Flag("host-controller-namespace", "The namespace on Host Cluster where install and controller jobs/deployments will be created (required for host aware mode, ignored otherwise)").String()
+		extManageTenantAPIServerHost     = extManageCmd.Flag("tenant-apiserver-host", "The Kubernetes apiserver Host for Tenant Kubernetes instance (required for host aware mode, ignored otherwise)").String()
+		extManageTenantAPIServerPort     = extManageCmd.Flag("tenant-apiserver-port", "The Kubernetes apiserver Port for Tenant Kubernetes instance (required for host aware mode, ignored otherwise)").String()
```

Comment:
Do we need API server and port if we're using a kubeconfig file? Can't those settings be in the kubeconfig?

Reply:
Yes, good point! I'll try that.
Originally I've implemented `Option 2` [here](https://github.com/crossplaneio/crossplane/blob/3a2671c46a98a8a46fbe90ff6c1f8c7b5b0a688f/design/one-pager-host-aware-stack-manager.md#host-cluster-access),  and missed this while switching to `Option 1`.

Reply:
Just realized another reason to get API server and port seperately. 
We need to set these parameters for crossplane deployment as well to configure it against tenant apiserver and we don't pass kubeconfig file to it at all: https://github.com/turkenh/crossplane/blob/host-aware-sm-onepager/design/one-pager-host-aware-stack-manager.md#configuring-a-pod-against-tenant-instance-api-server

I believe getting these two params separately would be better instead of passing kubeconfig to crossplane just for this purpose.